processor/tailsamplingprocessor/internal/sampling/numeric_tag_filter.go

```diff
@@ -41,9 +43,13 @@ func (naf *numericAttributeFilter) Evaluate(_ context.Context, _ pcommon.TraceID
 		if v, ok := span.Attributes().Get(naf.key); ok {
 			value := v.Int()
 			if value >= naf.minValue && value <= naf.maxValue {
-				return true
+				return !(naf.invertMatch)
+
 			}
 		}
-		return false
+		if !naf.invertMatch {
```

Comment:
Can't you just return the value of the invertMatch attribute?

```suggestion
		return naf.invertMatch
```